data/json/vehicles/cars.json

```diff
@@ -713,6 +713,81 @@
+      { "x": 0, "y": 0, "part": "vehicle_alarm" },
+      { "x": 0, "y": 0, "part": "horn_car" },
+      { "x": 0, "y": 0, "part": "roof" },
+      { "x": 0, "y": 1, "part": "frame_vertical_2" },
+      { "x": 0, "y": 1, "part": "seat" },
+      { "x": 0, "y": 1, "part": "seatbelt" },
+      { "x": 0, "y": 1, "part": "roof" },
+      { "x": 0, "y": -1, "part": "xlframe_vertical" },
+      { "x": 0, "y": -1, "part": "door" },
+      { "x": 0, "y": 2, "part": "xlframe_vertical" },
+      { "x": 0, "y": 2, "part": "door" },
+      { "x": 1, "y": 0, "part": "xlframe_horizontal" },
+      { "x": 1, "y": 0, "part": "windshield" },
+      { "x": 1, "y": 1, "part": "xlframe_horizontal" },
+      { "x": 1, "y": 1, "part": "windshield" },
+      { "x": 1, "y": -1, "part": "xlframe_vertical" },
+      { "x": 1, "y": -1, "part": "windshield" },
+      { "x": 1, "y": 2, "part": "xlframe_vertical" },
+      { "x": 1, "y": 2, "part": "windshield" },
+      { "x": 2, "y": 0, "part": "xlframe_horizontal" },
+      { "x": 2, "y": 0, "part": "xlhalfboard_horizontal" },
+      { "x": 2, "y": 0, "part": "engine_electric_super" },
+      { "x": 2, "y": 0, "part": "storage_battery" },
+      { "x": 2, "y": 1, "part": "xlframe_horizontal" },
+      { "x": 2, "y": 1, "part": "storage_battery" },
+      { "x": 2, "y": 1, "part": "xlhalfboard_horizontal" },
+      { "x": 2, "y": -1, "part": "xlframe_nw" },
+      { "x": 2, "y": -1, "part": "xlhalfboard_nw" },
+      { "x": 2, "y": -1, "part": "wheel_steerable" },
+      { "x": 2, "y": -1, "part": "headlight" },
+      { "x": 2, "y": 2, "part": "xlframe_ne" },
+      { "x": 2, "y": 2, "part": "xlhalfboard_ne" },
+      { "x": 2, "y": 2, "part": "wheel_steerable" },
+      { "x": 2, "y": 2, "part": "headlight" },
+      { "x": -1, "y": 0, "part": "frame_vertical" },
+      { "x": -1, "y": 0, "part": "storage_battery" },
+      { "x": -1, "y": 0, "part": "trunk" },
+      { "x": -1, "y": 0, "part": "roof" },
+      { "x": -1, "y": 1, "part": "frame_vertical" },
+      { "x": -1, "y": 1, "part": "storage_battery" },
+      { "x": -1, "y": 1, "part": "trunk" },
+      { "x": -1, "y": 1, "part": "roof" },
+      { "x": -1, "y": -1, "part": "xlframe_vertical" },
+      { "x": -1, "y": -1, "part": "xlhalfboard_vertical" },
+      { "x": -1, "y": 2, "part": "xlframe_vertical" },
+      { "x": -1, "y": 2, "part": "xlhalfboard_vertical" },
+      { "x": -2, "y": -1, "part": "xlframe_horizontal" },
+      { "x": -2, "y": -1, "part": "xlhalfboard_sw" },
+      { "x": -2, "y": -1, "part": "wheel_slick_steerable" },
+      { "x": -2, "y": 0, "part": "xlframe_horizontal" },
+      { "x": -2, "y": 0, "part": "door_trunk" },
+      { "x": -2, "y": 1, "part": "xlframe_horizontal" },
+      { "x": -2, "y": 1, "part": "door_trunk" },
+      { "x": -2, "y": 2, "part": "xlframe_horizontal" },
+      { "x": -2, "y": 2, "part": "xlhalfboard_se" },
+      { "x": -2, "y": 2, "part": "wheel_slick_steerable" }
```

Comment:
Nice!  Good use of slick wheels.

But not all the of the frames are xlframes, and a racing car should only have extra-light frames.

Reply:
It was originally a literal copy-paste of electric sports car where I just changed the motor to use the super motor. Which means another PR might need to change sports car to have only xlframes.

I implemented your suggestions, but I have one question what's the point of having only steerable wheels, shouldn't we have only steerable wheel at the front ?